### PR TITLE
hokus: Add version 0.6.1

### PR DESCRIPTION
**Pull request by niheaven**
https://www.hokuscms.com

https://github.com/julianoappelklein/hokus

### CODE REVIEW

File: bucket/hokus.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    "shortcuts": [
+        [
+            "Hokus CMS.exe",
+            "Hokus CMS"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/julianoappelklein/hokus/releases",
+        "regex": "tree/([\\d.]+)"
```
Comment (Ash258) [inline bucket/hokus.json:26]: You can go with classic github

```suggestion
        "github": "https://github.com/julianoappelklein/hokus"
```

Comment (niheaven) [reply bucket/hokus.json:26]: There isn't stable (or Latest) release now, so `github` doesn't work.

Comment (Ash258) [reply bucket/hokus.json:26]: It will be redirected.

**Pull request merged**